addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -61,23 +60,23 @@ var MassMailingFieldHtml = FieldHtml.extend({
 
         var $editable = this.wysiwyg.getEditable();
         await this.wysiwyg.cleanForSave();
-        return this.wysiwyg.saveModifiedImages(this.$content).then(function () {
-            self._isDirty = self.wysiwyg.isDirty();
-            self._doAction();
+        return this.wysiwyg.saveModifiedImages(this.$content).then(async () => {
```

Comment:
I think this comes from a validation comment so I'm sorry but I'l going to ask you the opposite, please revert to the "async function ()" code.

Why? Because this changes a lot of lines here below and it makes it hard to see the actual fix.
We usual don't do "linting" during bug fixes, to keep a minimal diff.

Also, it modifies the line with ".html", requiring us to do an security override for the mergebot, which could be avoided.